Automattic/vip-go-mu-plugins
# WP-CLI: User Cleanup command

## Description

Adds a new WP-CLI command which will allow us to revoke user permissions by email. This is useful when we need to handle user departure from an internal team or a customer.

TODO:
- [x] Add confirmation step
- [x] Add output to WP-CLI command

## Changelog Description

### New WP-CLI Command: User Cleanup

We've introduced a new command to make it easier to offboard users from sites. Given one or more email address, permissions (including super admin) can be removed from users matching those email addresses.

```
vip @<app>.<env> -- wp vip user cleanup --emails="test@example.com,also@example.com"
```

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
1. Generate several users using `wp user generate`
1. Run the cleanup `wp vip user cleanup --emails="<comma-separated-emails>"`
1. Verify users are removed.
